db/migrate/20170209005256_add_display_name_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddDisplayNameToUsers < ActiveRecord::Migration[5.0]
+  def change
+    add_column :users, :display_name, :string
```

Comment:
Can we call this `twitch_display_name`? I regret calling `name` `name`, as adding other sign in methods later just makes that confusing.